app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        _, current_file, future_file = command.split()
```

Comment:
if command not equal to cp
https://github.com/mate-academy/py-copy-file/blob/master/checklist.md use checklist before send to re-review